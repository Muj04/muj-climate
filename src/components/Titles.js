import React from 'react';
const Titles = (props) =>{
  return (
    <div>
      <h1 class= "App">Weather App </h1>
      <p> Helps you find weather conditions in cities... </p>
    </div>
  )
}
export default Titles;