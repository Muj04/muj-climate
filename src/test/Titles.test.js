import React from 'react'; 
import Titles from '../components/Titles';  
import renderer from 'react-test-renderer'; 

test('renders correctly', () => {  
 const tree = renderer    
 .create(<Titles/>)     
	.toJSON();  
	 expect(tree).toMatchSnapshot();  
	}); 