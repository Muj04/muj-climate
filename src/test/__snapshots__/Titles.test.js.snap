// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1
    class="App"
  >
    Weather App 
  </h1>
  <p>
     Helps you find weather conditions in cities... 
  </p>
</div>
`;
